scripts/bert/index.rst

```diff
@@ -115,9 +117,19 @@ Finally, GluonNLP also supports the "`DistilBERT <https://arxiv.org/abs/1910.011
     words, valid_len = mx.nd.array([sample[0]]), mx.nd.array([sample[1]])
     seq_encoding, cls_encoding = model(words, valid_len);
 
+Finally, GluonNLP also suports Korean BERT pre-trained model, [KoBERT](https://github.com/SKTBrain/KoBERT), trained with Korean wiki dataset ('kobert_news_wiki_ko_cased').
+
+.. code-block:: python
+
+    !pip install -U https://github.com/dmlc/gluon-nlp/archive/v0.9.0.tar.gz
```

Comment:
Usually we don't include pip install gluon-nlp line here.

Reply:
I will remove the line and do PR again.